tools/opensnoop.bt

```diff
@@ -41,7 +35,12 @@
  * 08-Sep-2018	Brendan Gregg	Created this.
  */
 
-config = { missing_probes=warn }
+#define MAX_PATH_DEPTH	35
+
+config = {
+	missing_probes=warn;
+	unstable_macro=enable;
```

Comment:
As mentioned in the other PR, I'd rather not use unstable features in these scripts for now.